select_statements.sql

```diff
@@ -31,89 +31,60 @@ UPDATE vehicle
 SET stationName = "Linköping station"
 WHERE registrationNumber = "CAG461";
 
-UPDATE vehicle
-SET stationName = "Lund station"
-WHERE registrationNumber = "AAD123";
-
-UPDATE vehicle
-SET stationName = "Lund station"
-WHERE registrationNumber = "AAC223";
-
--- Select all cars
-SELECT v.registrationNumber as Registreringsnummer, vc.name as Fordonskategori, v.stationName as Station
-FROM vehicle as v
-INNER JOIN vehicle_category as vc ON v.vehicleCategoryId = vc.vehicleCategoryId
-ORDER BY Station, FordonsKategori;
-
-
 -- Select all vacant cars for a station
 SELECT v.registrationNumber as Registreringsnummer, vc.name as Fordonskategori
-FROM vehicle as v
+FROM green_rental.vehicle as v
 LEFT JOIN booking_details as bd ON v.registrationNumber = bd.registrationNumber
 LEFT JOIN booking as b ON bd.bookingNumber = b.bookingNumber
 INNER JOIN vehicle_category as vc ON v.vehicleCategoryId = vc.vehicleCategoryId
-WHERE v.stationName = "Uppsala station" AND (b.endDate < "2024-04-13" OR b.endDate IS NULL)
-ORDER BY Fordonskategori;
+WHERE v.stationName = "Uppsala station" AND (b.endDate < "2024-04-13" OR b.endDate IS NULL);
 
 -- Select all vacant cars for all stations
 SELECT v.registrationNumber as Registreringsnummer, vc.name as Fordonskategori, v.stationName as Station
-FROM vehicle as v
+FROM green_rental.vehicle as v
 LEFT JOIN booking_details as bd ON v.registrationNumber = bd.registrationNumber
 LEFT JOIN booking as b ON bd.bookingNumber = b.bookingNumber
 INNER JOIN vehicle_category as vc ON v.vehicleCategoryId = vc.vehicleCategoryId
-WHERE b.endDate < "2024-04-13" OR b.endDate IS NULL
-ORDER BY Station, Fordonskategori;
+WHERE b.endDate < "2024-04-13" OR b.endDate IS NULL;
 
 -- Select all vacant cars for "my" station of a certain category
-SELECT v.registrationNumber, v.stationName FROM vehicle as v
+SELECT v.registrationNumber, v.stationName FROM green_rental.vehicle as v
```

Comment:
I removed `green_rental.` from all statements to make them easier to read in the report.